dxp/code_review.md

```diff
@@ -98,3 +98,84 @@ As before, we can see:
 
 -   Reason (own activity) from `X-GitHub-Reason` or the CC address of `your_activity@noreply.github.com`.
 -   GitHub user account and email address.
+
+## Keeping the [liferay-frontend/liferay-portal](https://github.com/liferay-frontend/liferay-portal) repo up-to-date
+
+As explained in [this Slack thread](https://liferay.slack.com/archives/G3JBR21HA/p1597765484336300), we have a [cron](https://en.wikipedia.org/wiki/Cron) job running to keep the team repo in sync with the upstream [liferay/liferay-portal](https://github.com/liferay/liferay-portal) repo.
+
+> :information_source: If the sync job stalls and the repo gets stale, anybody who is a repo collaborator (ie. anybody on the team) is able to push `master` to get it back up-to-date at any time. Additionally, anybody with push privileges can run the cron job (although so far, we've only been running it on one always-connected developer laptop).
```

Comment:
Here's to that fiery-eyed always-connected developer laptop 🍻

Reply:
Should I also mention that it is always connected because I am working *literally* 24/7?

`#WorkLifeBalance`

![working-hard](https://user-images.githubusercontent.com/7074/96558299-c3514680-12bb-11eb-80bf-1ffc636a0694.gif)

Reply:
> Should I also mention that it is always connected because I am working literally 24/7?

No, we're not allowed to say that. It's bad PR.

Reply:
> bad PR

![get-it](https://user-images.githubusercontent.com/7074/96559413-4cb54880-12bd-11eb-9ac6-2f5d0f90c16b.gif)